### DISCUSSION TITLE
希望影片

### DISCUSSION THREAD

**Discussion by hite385** [share121/inter-knot#42] [其他]
我这样的人渣，居然能来到绳网，这是何等的斯巴拉西啊
╯ 　乀
ヘ　 へ
　′
　﹀

**Comment by Tragedysaver1024** [comment]
这位先生你好，麻烦私聊一下，我真的是有点好奇您是怎么能在一座全封闭日日都有杀人案并且没法联系外界的学校内是怎么能登上绳网的

**Reply by hite385** [reply]
🤔
